src/Token.ts

```diff
@@ -0,0 +1,16 @@
+// @author Anush kumar N <anush.kumar@codingmart.com>
+
+// Collection of All tokens
+
+const TokenType = {
```

Comment:
Why not individually export each token? So at the call site it can be imported like

```
import { 
    dot, 
    integer, 
    ident,
} from './tokens';
```

Then the compiler and build system will be able to help us out.